src/routes/api/users.go

```diff
@@ -17,9 +18,12 @@ var usersController *controllers.UsersController
 func UsersRoute(db *gorm.DB) func(router fiber.Router) {
 	usersController = controllers.NewUsersController(db)
 	return func(router fiber.Router) {
+		// Public Routes
 		router.Post("/", registerUser)
 		router.Post("/login", loginUser)
-		router.Get("/:id", auth.MandatoryAuthMiddleware, getUserInfo)
+		router.Get("/:id", getUserInfo)
```

Comment:
the `get` path should also be private here

Reply:
fixed.